### PR TITLE
app-admin/elektra: Fix build w/ GCC 5.4.0, version bump to 0.8.16

### PR DESCRIPTION
**Pull request by a17r**
@xmw

### CODE REVIEW

**Comment by jaakristioja** [comment]
Elektra 0.8.17 has also been [released](http://doc.libelektra.org/news/e6153a39-c4bd-41c3-bc86-785d451eb6c5.html).

**Comment by monsieurp** [comment]
@a17r Hi. Could you file a bug report and ping @xmw there? He doesn't seem to be very interested in Github and your contribution will never make it into the tree using this route. Thanks.

**Comment by monsieurp** [comment]
ping?

**Comment by a17r** [comment]
Well, the bug exists... https://bugs.gentoo.org/show_bug.cgi?id=586054

File: app-admin/elektra/elektra-0.8.15.ebuild [hunk 1]
```diff
@@ -97,10 +100,10 @@ multilib_src_configure() {
...
 
 	mycmakeargs=(
+		"-DBUILD_PDF=OFF"
 		"-DBUILD_SHARED=ON"
 		"-DPLUGINS=${my_plugins}"
 		"-DTOOLS=${my_tools}"
-		"-DLATEX_COMPILER=OFF"
 		"-DTARGET_CMAKE_FOLDER=share/cmake/Modules"
 		$(multilib_is_native_abi && cmake-utils_use doc BUILD_DOCUMENTATION \
 			|| echo -DBUILD_DOCUMENTATION=OFF)
```
Comment (SoapGentoo) [inline app-admin/elektra/elektra-0.8.15.ebuild]: can we simply this please? `-DBUILD_DOCUMENTATION=$(multilib_is_native_abi && usex doc || echo no)`

Comment (SoapGentoo) [inline app-admin/elektra/elektra-0.8.15.ebuild]: `rm -rvf "${ED%/}"/` in `multilib_src_install_all` please

**Pull request closed without merging**